variables.tf

```diff
@@ -18,10 +18,15 @@ variable "code_commit_repo_branch" {
 }
 
 variable "code_commit_repo_name" {
```

Comment:
Should we change this variable name to just `repo_name` since it would be used in both the CodeCommit and CodeStar Connection cases?

Reply:
We could -- but that would break backward compatibility for anyone using an older version of the terraform module. Generally I would prefer it, but I didn't change it for that reason. Let me know what you think and I'll rename it.

Reply:
Yeah, we would have to release a new major version, but I think that renaming both the `code_commit` fields is better for the long term.